chatterbox/src/Main/Header/index.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import './index.css';
+import img1 from '../../assets/img1.png';
+import img2 from '../../assets/img2.png';
+import AppBar from 'material-ui/AppBar';
+import Toolbar from 'material-ui/Toolbar';
+import IconButton from 'material-ui/IconButton';
+import MenuIcon from 'material-ui-icons/Menu';
+import Menu, { MenuItem } from 'material-ui/Menu';
+import Button from 'material-ui/Button';
+
+const styles = theme => ({
+    root: {
+        height: ' 103px',
+        width: '1440px',
+        color: '#01B9BD'
+    },
+
+    flex: {
+        flex: 1
+    },
+    menuButton: {
+        marginLeft: -12,
+        marginRight: 20
+    },
+
+    images: {
+        borderRadius: 50
+    }
+});
+
+class MenuAppBar extends React.Component {
+    state = {
+        auth: true,
+        anchorEl: null
+    };
+
+    static propTypes = {
+        classes: PropTypes.object.isRequired
+    };
+
+    handleChange = (event, checked) => {
+        this.setState({ auth: checked });
+    };
+
+    handleMenu = event => {
+        this.setState({ anchorEl: event.currentTarget });
```

Comment:
Same as previous comment, no setstate, must be redux

Reply:
That's the current plan now.